extra-productivity/joplin/autobuild/prepare

```diff
@@ -0,0 +1,22 @@
+abinfo "Setting Locale Environment Variable for npm..."
+export LANG=en_US.UTF-8
```

Comment:
Is this really necessary?

Reply:
Yes. 

Ref: https://github.com/alfredopalhares/joplin-pkgbuild/issues/25

Reply:
Well we default to C.UTF-8.

Reply:
tested. this tweak doesn't need